app.js

```diff
@@ -35,7 +35,9 @@ app.use(favicon(path.join(__dirname, 'public', 'images', 'favicon.ico')));
 // default value for title local
 app.locals.title = 'Express - Generated with IronGenerator';
 
-const index = require('./routes/index');
-app.use('/', index);
+//const index = require('./routes/index');
+app.use("/", require("./routes/index"));
```

Comment:
The declaration of these app.use's are not necessary.
I would keep the app.use('/', index), and then put all the routes in that file.